spec/acceptance/cli/install_spec.rb

```diff
@@ -67,4 +68,23 @@
         'bundle install --gemfile=gemfiles/1.0.0.gemfile --jobs=1')
     end
   end
+
+  context "with full-index", parallel: true do
```

Comment:
A synonym for `parallel: true` is `:parallel`.

Reply:
Oh, nice, thanks! I hadn't thought of that.